lib/Model.js

```diff
@@ -298,11 +298,19 @@ Model.prototype.put = function(options, next) {
       options.overwrite = true;
     }
 
-    let toDynamoOptions = {updateTimestamps: true};
+    let toDynamoOptions = {
+      updateExpires: false,
```

Comment:
We don't need this, correct? It should just default to false by default without having this line.

Reply:
I understand your point but isn't it better to have the default set so it is visible through the system that it is actually set to something?

Reply:
@devotox I'm pretty sure the only reason to have it there is to increase readability and ease of understanding for developers in the future. Anytime we are comparing that value to `false` or `true`, `undefined` will behave just like `false`. So I don't think removing this is a bad thing. Especially since it looks like you did and all the tests are passing. If you believe there is an edge case that your tests haven't accounted for that this would break please write a test and if that test fails then yes we should totally add it back in.